pyvcloud/vcd/vdc.py

```diff
@@ -373,6 +373,42 @@ def list_edge_gateways(self):
                 })
         return edge_gateways
 
+    def get_nat_rules(self, edge_gateway_name):
+        """Request the nat rules of an edge gateway.
+
+        :param edge_gateway_name: (str): The name of the edge gateway.
+
+        :return: A list of :class:`lxml.objectify.StringElement' objects
+            representing the nat rules.
+
+        :raises: Exception: if there is no edge gateway with the provided name.
+        :raises: Exception: if there are no nat rules on the edge gateway
```

Comment:
I guess it's possible to have an edge gateway without any NAT rules configured on it.  Keeping that in mind, I don't think it's correct to throw an exception if no NAT rules are found on the edge gateway. Rather we should just return an empty list back.